### DISCUSSION TITLE
LDM Notes for April 26th, 2023

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#7166] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2023/LDM-2023-04-26.md

## Agenda

- Collection literals

**Comment by huoyaoyuan** [comment]
> Next, we looked at KVPs turning the collection literal into a Dictionary<> automatically.

I'm in favor of disabling the K:V syntax for List\<KVP\>, and only allow it for Dictionary<K, V>. It's a clear distinction to me: when treating KVP as a usual type, do not enable special syntax to it.
